### PR TITLE
En 5621 maligned fixes

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
changed order of elements in some structs in order to save some space caused by memory padding. Issues reported by maligned tool of golangci

didn't regenerate schema for capnp

### CODE REVIEW

File: data/block/metaBlock.go [hunk 1]
```diff
@@ -69,17 +69,17 @@ type ShardMiniBlockHeader struct {
...
 	ShardMiniBlockHeaders []ShardMiniBlockHeader `capid:"2"`
 	PrevRandSeed          []byte                 `capid:"3"`
 	PubKeysBitmap         []byte                 `capid:"4"`
 	Signature             []byte                 `capid:"5"`
-	TxCount               uint32                 `capid:"6"`
 	Round                 uint64                 `capid:"7"`
 	PrevHash              []byte                 `capid:"8"`
 	Nonce                 uint64                 `capid:"9"`
 	NumPendingMiniBlocks  uint32
+	ShardID               uint32 `capid:"0"`
```
Comment (AdoAdoAdo) [inline data/block/metaBlock.go]: remove the capids

File: data/block/block.go [hunk 2]
```diff
@@ -99,22 +99,22 @@ type Header struct {
...
-	MiniBlockHeaders       []MiniBlockHeader `capid:"12"`
-	PeerChanges            []PeerChange      `capid:"13"`
 	RootHash               []byte            `capid:"14"`
 	ValidatorStatsRootHash []byte            `capid:"15"`
 	MetaBlockHashes        [][]byte          `capid:"16"`
 	EpochStartMetaHash     []byte            `capid:"17"`
-	TxCount                uint32            `capid:"18"`
 	ReceiptsHash           []byte            `capid:"19"`
 	ChainID                []byte            `capid:"20"`
+	MiniBlockHeaders       []MiniBlockHeader `capid:"12"`
```
Comment (AdoAdoAdo) [inline data/block/block.go]: can you also remove all capids  as we anyway are going to drop capnproto for gogo protobufs and the ids are not ordered now.

File: data/block/metaBlock.go [hunk 3]
```diff
@@ -113,11 +112,12 @@ type MetaBlock struct {
 	RandSeed               []byte            `capid:"11"`
 	RootHash               []byte            `capid:"12"`
 	ValidatorStatsRootHash []byte            `capid:"13"`
-	TxCount                uint32            `capid:"14"`
 	MiniBlockHeaders       []MiniBlockHeader `capid:"15"`
 	ReceiptsHash           []byte            `capid:"16"`
 	EpochStart             EpochStart        `capid:"17"`
 	ChainID                []byte            `capid:"18"`
+	Epoch                  uint32            `capid:"1"`
```
Comment (AdoAdoAdo) [inline data/block/metaBlock.go]: can you remove the capids?

File: data/rewardTx/rewardTx.go [hunk 4]
```diff
@@ -11,10 +11,10 @@ import (
 // RewardTx holds the data for a reward transaction
 type RewardTx struct {
 	Round   uint64   `capid:"1" json:"round"`
```
Comment (AdoAdoAdo) [inline data/rewardTx/rewardTx.go]: can you remove the capids?

File: data/block/metaBlock.go (hunk 1 above)
Comment (bogdan-rosianu) [reply data/block/metaBlock.go]: done

File: data/block/block.go (hunk 2 above)
Comment (bogdan-rosianu) [reply data/block/block.go]: done

File: data/block/metaBlock.go (hunk 3 above)
Comment (bogdan-rosianu) [reply data/block/metaBlock.go]: yes

File: data/rewardTx/rewardTx.go (hunk 4 above)
Comment (bogdan-rosianu) [reply data/rewardTx/rewardTx.go]: yes

**Pull request merged**